### PR TITLE
New extension - Twitter trendscast 

### PR DESCRIPTION
**Pull request by pradeepb28**
## Description

A new extension is added in this PR. The purpose of this extension is able to checkout the what's happening on Twitter - be it top trends, news, top tweets from people you follow etc. 

## Screencast

CleanShot 2022-03-03 at 12 11 17

CleanShot 2022-03-03 at 12 11 31

## Checklist

- [Y] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [Y] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [Y] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [Y] I checked that files in the `assets` folder are used by the extension itself
- [Y] I checked that assets used by the `README` are placed in the `media` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @pradeepb28 👋 

Thanks for your contribution, I while testing it, I have a question:

- Both `What's happening around you?` and `What's trending around you?` is not working for me, I just end here:

DHZh8dFax4RZKzYI

Can you look into this?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by pradeepb28** [comment]
@pernielsentikaer It seems to be working on my side 
here is the [video link](https://photos.app.goo.gl/jQziZ7hsYJohuLVC6)

I assume it might not be working because Twitter might be using different URL for a non english localization?

May I know what language you using on Twitter and what browser are you using?

**Comment by pradeepb28** [comment]
I have tested this specifically on Danish language on Twitter, here is the [video link](https://photos.app.goo.gl/1vMhbUi6yNMgQwu69)

**Comment by pernielsentikaer** [comment]
Hi @pradeepb28 👋 

I have now cleared my browser and logged in again, same result – see this: https://cln.sh/eIiQzr

If it's easier, I'm available the most of the time in our [Slack](https://raycastcommunity.slack.com/) - feel free to catch me in there 😄

**Comment by pradeepb28** [comment]
@pernielsentikaer I've removed those two shortcuts that are not working on your side.

File: extensions/twitter-trendscast/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "name": "twitter-trendscast",
+  "title": "Twitter Trendscast",
+  "description": "With these simple shortcuts, you can keep up with the latest trends from people you follow or any other trends on Twitter.",
+  "icon": "ext-icon.png",
+  "author": "pradeepb28",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "twitter-trendscast",
+      "title": "Twitter Trendscast",
```
Comment (pernielsentikaer) [inline extensions/twitter-trendscast/package.json:12]:
```suggestion
      "title": "Show Trendscast",
      "subtitle": "Twitter",
```
What do you think about this? I'm not sure if it's better than it is right now

**Review by pernielsentikaer — COMMENTED** [review]
Hi @pradeepb28 👋 

I'm available for more testing to find out why it's not working, but for now, let's get it merged 🚀 

I just have a single correction to discuss before merging 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/twitter-trendscast/package.json (hunk 1 above)
Comment (pradeepb28) [reply extensions/twitter-trendscast/package.json:12]: I've updated something similar with few changes "Show trends from twitter" @pernielsentikaer

**Review by pernielsentikaer — APPROVED** [review]
Thanks for the update @pradeepb28 🔥 

I was nice chatting with you in Slack, let me know if you need anything 💪 

Approved! 🚀

**Comment by fe9lix** [comment]
@pradeepb28 @pernielsentikaer fyi, fixed the title case: https://github.com/raycast/extensions/commit/59c984480836b959c537f8c6ceeea5281020a365

**Pull request merged**